### ISSUE TITLE
Fee confirmation doesn't allow correction, as intended

### ISSUE THREAD

**Issue by bitcoinhodler** [GlacierProtocol/GlacierProtocol#21]
During the withdrawal process, the interactive fee calculation is designed to calculate the total fee based on the user's input sat/B, then allow the user to correct this if the total fee looks incorrect. However, if the user answers No at the confirmation, glacierscript simply repeats the calculation using the same fee rate, instead of prompting the user for a new fee rate as the code was clearly intended to do.

# Expected behavior
After the total fee is shown and the user is prompted "Confirm? (y/n)", entering No should again prompt the user to enter the desired fee rate.

# Actual behavior
When the user enters No at the confirmation, the same total fee is shown again with another confirmation prompt. The user has no chance to change the fee rate.

# Impact
Minor annoyance. The only way to make a correction to the fee rate is to Ctrl-C and run the script again.

# Root cause
The problem is inside function `get_fee_interactive`:
```
        if not fee_basis_satoshis_per_byte:
            print "\nEnter fee rate."
            fee_basis_satoshis_per_byte = int(raw_input("Satoshis per vbyte: "))
```
If the user enters No at the confirmation, this code runs again, but this time the variable `fee_basis_satoshis_per_byte` already has a value (as entered by the user the first time).

# Solution
When the user enters No at the confirmation, the code should clear out the value of `fee_basis_satoshis_per_byte` so the prompt will happen on the second pass.

**Comment by bitcoinhodler** [comment]
A similar problem occurs when the calculated fee exceeds the hard-coded MAX_FEE, though in this case the script loops forever and must be Ctrl-C'd to stop. I'll add a fix for that to the same PR.

**Comment by bitcoinhodler** [comment]
This is fixed in v0.92
